kpconv_torch/datasets/S3DIS.py

```diff
@@ -82,7 +82,7 @@ def __init__(
         self.use_potentials = use_potentials
```

Comment:
Why this choice only at some lines of the code and not everywhere ?

Reply:
Which choice do you refer to? Is it the transition towards `pathlib.Path`?

Well, we introduced `pathlib.Path` through the CLI, and I did the same choice in functional tests. That being said, some instructions in the code expect `str` variables: this PR contains only the required changes to make the tests work, by changing the corresponding instructions.

By the way, the usage of `pathlib.Path` should be generalized, but I prefer not to include this into the current PR.

Reply:
Yes, this choice. OK to postpone this transition to another PR.